docs/getting-started/installation.md

```diff
@@ -144,7 +144,7 @@ If it outputs `insufficient privileges`, you need to add your user to the
 `kvm` group. You can do this with:
 
 ```
-adduser "$USER" kvm
+sudo usermod -a -G kvm "$USER"
```

Comment:
May be this is distro specific? @adamkpickering , Does this command proposal/change makes sense to you?

Reply:
The command is wrong in all distros. `adduser` is for creating an user, not for adding the user to a group. (https://linux.die.net/man/8/adduser)

Reply:
Looks like it is distro specific. On pop OS 20.04, `adduser` works in the way it is currently documented. On openSUSE Leap 15.3 it doesn't appear to be installed by default, but appears to be available. On Fedora 36 it is (confusingly) symlinked to `useradd`.

@drzymalanet if you look under the Synopsis section, you'll see `useradd`. Which is very confusing. Those docs must be written up for distros based on Red Hat or something.

Either way, `usermod` is more cross-platform. I tested it on the above three distros and it works on all.

Reply:
Oh that's right, I am using rocky linux that's why `adduser` did not work for me. Thank you for testing this @adamkpickering I appreciate. By the way - does it really work on fedora 36? As said I run rocky and on k3d there are iptables/nftables issues and in rancher desktop qemu crashes.

Reply:
Rancher Desktop should work on Fedora 36. If it doesn't, please file a bug... or submit a PR :). But if RD has an issue in rocky, that issue is likely to be there in Fedora as well.